model/emoji_test.go

```diff
@@ -20,9 +20,8 @@ func TestEmojiIsValid(t *testing.T) {
 		Name:      "name",
 	}
 
-	if err := emoji.IsValid(); err != nil {
-		t.Fatal(err)
-	}
+	err := emoji.IsValid()
+	require.Nil(t, err)
```

Comment:
We don't need `err` variable
```suggestion
	require.Nil(t, emoji.IsValid())
```

Reply:
Yeah, you are right.